REFERENCE.md

```diff
@@ -127,6 +133,24 @@ wireguard::interface {'as3668-2':
   mtu                   => 1412,
 ```
 
+##### create a wireguard interface with multiple peers
+
+```puppet
+wireguard::interface {'wg0
+  dport     => 1338,
+  addresses => [{'Address' => '192.168.123.1/24'},],
```

Comment:
I'd consider using the RFC example ranges. https://en.wikipedia.org/wiki/Reserved_IP_addresses lists TEST-NET-1 (`192.0.2.0/24`), TEST-NET-2 (`198.51.100.0/24`) and TEST-NET-3 (`203.0.113.0/24`).

Also, slight coding style:
```suggestion
  addresses => [{'Address' => '192.168.123.1/24'}],
```